usecases/modules/modules.go

```diff
@@ -677,7 +677,7 @@ func (p *Provider) VectorFromInput(ctx context.Context,
 	for _, mod := range p.GetAll() {
 		if p.shouldIncludeClassArgument(class, mod.Name(), mod.Type()) {
 			if vectorizer, ok := mod.(modulecapabilities.InputVectorizer); ok {
-				cfg := NewClassBasedModuleConfig(class, mod.Name())
+				cfg := NewClassBasedModuleConfig(class, mod.Name(), "")
```

Comment:
you are missing `tenant` here, this method is used by `hybrid` you need to expand `VectorFromInput` method to be also able to pass tenant